txpipe/random_cats.py

```diff
@@ -168,7 +175,7 @@ def run(self):
             # Generate the random points in each pixel
             ndone = 0
 
-            nvertex = len(vertices)
+            nvertex = npix #number of verticies is the same as number of pixels
```

Comment:
Just trying to make sure I understand, is this change significant?

Reply:
no, it changes nothing 
len(vertices) == npix
We can switch it back if that's preferred